chipy_org/apps/meetings/templates/meetings/emails/meeting_topic_submitted.html

```diff
@@ -5,4 +5,4 @@
 <p>Presenters: {% for presenter in topic.presentors.all %}{{ presenter.name }} ({{ presenter.email }}) - {{ presenter.user.id }} {% endfor %}</p>
 <p>Meeting: {{ topic.meeting }}</p>
 <p>Length: {{ topic.length }}</p>
-<p>Description: {{ topic.description }}</p>
+<p>Description: {{ topic.description|safe }}</p>
```

Comment:
topic.description is generated from user input and should not be marked "safe".  Please look for a way to sanitize (remove bad html tags such as script) from the user input.